create-collector.js

```diff
@@ -40,7 +40,7 @@ copyFiles(templateDir, collectDir)
 console.log('populating writing cloud formation template')
 const fileContents = fs.readFileSync('collectors/template/cfn/collector.template.template', 'utf8')
 const replacedContents = populateTemplate(fileContents)
-const templatePath = collectDir + `/cfn/${type}_collector.template`
+const templatePath = collectDir + `/cfn/${type}-collector.template`
```

Comment:
looks like we need to change https://github.com/alertlogic/paws-collector/pull/108/files#diff-5031673c0a89d4324ddbe27623740ab7R68 as well